abstract/abstract.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Processing potential fields data with Fatiando a Terra'
```

Comment:
It looks like this one has the wrong content for the abstract -- this one is Santi's abstract :) The content is here: https://docs.google.com/document/d/1ycMuS99pGvG4MWlw0DBzrCuVJQjLTazVtXbkc2Ob7BI/edit

Would you mind updating @tetikshajain?

(Also, it looks like there are 2 pull requests open on this repo, just let me know which one to pay attention to, thanks!)